### ISSUE TITLE
90multipath: mpathconf script is only included in RH distributions

### ISSUE THREAD

**Issue by xosevp** [dracutdevs/dracut#1668]
```
$ git grep mpathconf
modules.d/90multipath/module-setup.sh:        mpathconf \
modules.d/90multipath/module-setup.sh:            mpathconf "${_args[@]}" --outfile "${initdir}"/etc/multipath.conf
modules.d/90multipath/multipathd-configure.service:ExecStart=/usr/sbin/mpathconf --enable
modules.d/90multipath/multipathd.sh:    mpathconf --enable
```
mpathconf only exists in Fedora/RH distributions and derivative ones(CentOS, Oracle, ...).
Source: https://src.fedoraproject.org/rpms/device-mapper-multipath/blob/rawhide/f/0018-RH-add-mpathconf.patch

@mwilck @bmarzins

**Comment by johannbg** [comment]
@xosevp given that the patch you are referring to is for device mapper multipath, how is this an bug in dracut?

**Comment by xosevp** [comment]
A bug in dracut.

**Comment by mwilck** [comment]
Unless I'm mistaken, this would be solved by #1664.

**Comment by LaszloGombos** [comment]
Closing based on @mwilck 's last comment.
